src/wasm/wasm-validator.cpp

```diff
@@ -549,6 +549,10 @@ void FunctionValidator::noteBreak(Name name,
 }
 void FunctionValidator::visitBreak(Break* curr) {
   noteBreak(curr->name, curr->value, curr);
+  if (curr->value) {
+    shouldBeTrue(
+      curr->value->type != none, curr, "break value must be a concrete type");
```

Comment:
text is misleading, as unreachable etc. is ok too

Reply:
Thanks, will fix.